app/models/metadata_presenter/evaluate_conditions.rb

```diff
@@ -3,25 +3,35 @@ class EvaluateConditions
     include ActiveModel::Model
     attr_accessor :service, :flow, :user_data
 
+    AND_CONDITIONS = %w[IF AND].freeze
```

Comment:
am ok with these being lower case as well. save having to upcase the condition_type further down.

i'm not sure 'if' counts as an 'and' condition.

Reply:
Maybe I should rename this constant to something else. In my view, we should use "if" only when we have a single criteria for the condition.

Reply:
i guess it depend what this will look like with the OR. we could go without the constant for now until we get to the OR?

Reply:
Ok, so I will change the if to the other way around to use the OR instead.

Reply:
Changed to use the other way around ♻️